src/etcd_client.rs

```diff
@@ -36,9 +30,7 @@ impl EtcdClient {
     async fn get_list_async<T: DeserializeOwned>(&self, prefix: &str) -> anyhow::Result<Vec<T>> {
         let req = etcd_rs::RangeRequest::new(etcd_rs::KeyRange::prefix(prefix));
         let mut resp =
-            self.etcd_rs_client.kv().range(req).await.map_err(|e| {
-                anyhow!("failed to get RangeResponse from etcd, the error is: {}", e)
-            })?;
+            self.etcd_rs_client.kv().range(req).await?;
```

Comment:
add `.context(format!("failed to get RangeResponse from etcd for RangeRequest={:?}", req))?;`

Reply:
The request doesn't implement `std::fmt::Debug`